src/Game/Game.elm

```diff
@@ -232,7 +232,7 @@ viewStats : Model -> Html Msg
 viewStats ({ hero } as model) =
     div []
         [ div [] [ text "Stats:" ]
-        , div [] [ text <| "HP: " ++ (Stats.printHP hero.stats) ]
+        , div [ class (Stats.getHPClass hero.stats) ] [ text <| "HP: " ++ (Stats.printHP hero.stats) ]
```

Comment:
I'd consider moving this whole div into Stats.elm and at this point, just doing `Stats.viewHP hero.stats`
Then in Stats.elm replace the printHP/SP with viewHP/SP